plutus-metatheory/src/Type.lagda.md

```diff
@@ -81,7 +81,10 @@ scope.
 A type is either a type variable, a pi type, a function type, a
 lambda, an application, an iso-recursive type `μ`, a size, or a type
 constant (base type). Note that recursive types range over an
-arbitrary kind `k` which goes beyond standard iso-recursive types.
+arbitrary kind `k` which goes beyond standard iso-recursive types. Also note that `Π`,
```

Comment:
What's the little title/layout table at the start of the document?  It looks as if it's out of place.

Later: no, I see that they all have that.

Reply:
It is supposed to set up stuff in the Markdown - presumably it makes the title tags for the HTML header? I suspect it is ignored by what we are currently using to make docs, but it is probably good to have it there.